README.rst

```diff
@@ -30,27 +30,27 @@ You're all set! Retrieve the list of available indices by running in your termin
 
 ::
 
-    run-func search getIndices
+    run-func index.js getExistingIndices
 
 
 How to use
 ----------
 
-1. Make sure that you have `full_format_recipes.json` at the root level of the repository and run command to inject data::
+1. Make sure that you have `full_format_recipes.json` at the root level of the repository and run command to index data:
 
-    run-func index injectData
+    run-func index.js indexData
```

Comment:
See https://github.com/Aiven-Labs/demo-open-search-node-js/blob/main/index.js#L16

Reply:
...and I got a 429 error (Too Many Requests) - maybe because I'm using a hobbyist plan?